src/cases/cases.module.ts

```diff
@@ -70,6 +69,7 @@ import { CreateCaseEventTriggerResolver } from './resolvers/create-case-event-tr
 // from services
 import * as fromServices from './services';
 import { effects, reducers } from './store';
+import { RpxTranslationModule } from 'rpx-xui-translation';
```

Comment:
@MateiHMCTS - please re-order the imports

Reply:
I think it's already ordered as '.' is before 'r'

Reply:
Hmm... interesting. In VSCode, for example, `rxjs` comes before `./users.effects` - can leave for now although the pipeline linter may flag it

Reply:
Nothing from yarn lint